std/internal/cstring.d

```diff
@@ -41,18 +41,6 @@ module std.internal.cstring;
 import std.range;
 import std.traits;
 
-version(unittest)
-@property inout(C)[] asArray(C)(inout C* cstr) pure nothrow @nogc @trusted
```

Comment:
Putting a trusted function like this in the global namespace is not a good idea.